### DISCUSSION TITLE
Crash after playing a beatmap throws unknown exception

### DISCUSSION THREAD

**Discussion by adryzz** [ppy/osu#16788] [Q&A]
I was playing a map (namely [this](https://osu.ppy.sh/beatmapsets/285795#mania/645430) map), and at the results screen, as it was saving the score to the db the game froze and crashed.
After backing up the logs ([database.log](https://github.com/ppy/osu/files/8008301/database.log) and [runtime.log](https://github.com/ppy/osu/files/8008302/runtime.log)), i restarted the game and it started the db from scratch (ignore the bass error)

![image](https://user-images.githubusercontent.com/46694241/152648803-ed1f18f4-28b1-493f-8a35-11d7b7fd4d14.png)

If the db is needed i'll send it over (both pre-realm migration and post migration).

> Running osu 2022.205.0.0 on .NET 5.0.6
> Environment: Linux (Unix 5.16.5.1), 12 cores

**Comment by peppy** [comment]
It's very weird that the next startup failed. Could you please confirm that restoring the backup still gets the same startup error, and if so, upload your database?

**Reply by adryzz** [reply]
Restoring the backup does not get the same startup error. There are no maps tho. Not sure if these are the correct ones, also because there are no other backups.
![image](https://user-images.githubusercontent.com/46694241/152652361-d10372b7-6238-45af-93fe-b323b166a629.png)

**Reply by peppy** [reply]
There should be a backup with "corrupt" in the name. Probably not in the backup folder itself.
